.travis.yml

```diff
@@ -18,18 +17,25 @@ install:
   - rustup toolchain install $RUST
   - rustup default $RUST
   - rustup target add arm-unknown-linux-gnueabihf
+  - rustup target add armv7-unknown-linux-gnueabihf
   - rustc -V
   - cargo -V
   - cp scripts/config ~/.cargo/config
   - git clone https://github.com/raspberrypi/tools.git ~/pi-tools
   - cp scripts/gcc-sysroot ~/pi-tools/arm-bcm2708/gcc-linaro-arm-linux-gnueabihf-raspbian-x64/bin
   - chmod +x ~/pi-tools/arm-bcm2708/gcc-linaro-arm-linux-gnueabihf-raspbian-x64/bin/gcc-sysroot
+  - export PATH="$HOME/pi-tools/arm-bcm2708/gcc-linaro-arm-linux-gnueabihf-raspbian-x64/bin:$PATH"
```

Comment:
So these steps are still necessary with this process? In that case I think they should be mentioned in `README.md`. Otherwise I would love to see them removed!

Reply:
Depends on where you get the toolchain from. On my setup I use [this](https://aur.archlinux.org/packages/arm-linux-gnueabihf-gcc/) from AUR, so I don't need to do this, but it's very Arch Linux-specific. If you are compiling on the RPi itself I'm guessing that's a different process as well. Should be up to the user.

Reply:
All of this is for making it prefer arm libraries over x86 ones when cross compiling. Does your AUR package just do it automatically?